### PR TITLE
[Feature] Allow users to personalize layouts.

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska @hcarneiro 

## Issue
https://github.com/Fliplet/fliplet-studio/issues/6225

## Description
Allow users to personalize layouts.

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -38,6 +38,11 @@ export const state = {
 export function handleWidgetData(data) {
   state.widgetData = data
 
+  if (/:/.test(data.widgetPackage)) {
+    state.widgetData.widgetLayout = Fliplet.Widget.getAttributes(data.widgetId)
+    state.widgetData.widgetPackage = data.widgetPackage.split(':')[0]
```
Comment (hcarneiro) [inline src/store/index.js:43]: The package won't have `:`. this will just be the normal package name.

Comment (hcarneiro) [reply src/store/index.js:43]: @YaroslavOvdii We won't be passing the `:layoutName` anymore via query. That's why you get the layout via `Fliplet.Widget.getAttributes(data.widgetId)`.

In my instructions I said: `Save the package only as currently is`

Comment (hcarneiro) [reply src/store/index.js:43]: This means that you don't need to reassign `state.widgetData.widgetPackage`. Also means that `Fliplet.Widget.getAttributes(data.widgetId)` will return a value or nothing. @squallstar needs to confirm what will return if a component doesn't have an attribute to get.

Comment (squallstar) [reply src/store/index.js:43]: Most likely we will return `undefined` when the attribute is not found.

Comment (YaroslavOvdii) [reply src/store/index.js]: @hcarneiro Fixed

**Comment by hcarneiro** [comment]
Tis shouldn't be merged until @squallstar does his backend changes. Otherwise this can't be tested.

**Pull request merged**